capstone/src/main/java/com/step902020/capstone/IndividualController.java

```diff
@@ -169,17 +169,24 @@ public RedirectView deleteSavedOrganization(
   /**
    * gets a combined list of events from the user's saved list of events and events from all the
    * saved organizations
-   * @return Iterable containing Events
+   * @param email current user email from IAP header
+   * @return list of events
    * @throws IOException
    */
   @GetMapping("get-calendar-events")
-  public Iterable<Event> getCalendarEvents() throws IOException {
-    /* full implementation blocked by the previous PR (because I changed some properties into references )
-     and those changes are not in this branch */
-    // get saved events
-    // get a list of organizations
-    // for each of the organizations get their list of saved events
-
-    return this.eventRepository.findAll();
+  public List<Event> getCalendarEvents(
+          @RequestHeader("X-Goog-Authenticated-User-Email") String email) throws IOException {
```

Comment:
Same, use `CurrentUser` object instead. Otherwise this won't be testable locally.